source/shared/cpp/AdaptiveCardsSharedModel/AdaptiveCardsSharedModelUnitTest/EverythingBagel.json.orig

```diff
@@ -0,0 +1,371 @@
+					"isMultiline": false,
+					"label": "Input.Text_label",
+					"maxLength": 10,
+					"placeholder": "Input.Text_placeholder",
+					"style": "text",
+					"value": "Input.Text_value",
+					"spacing": "small",
+					"isRequired": false,
+					"regex": "([A-Z])\\w+",
+					"inlineAction": {
+						"type": "Action.Submit",
+						"iconUrl": "https://adaptivecards.io/content/cats/1.png",
+						"title": "Input.Text_Action.Submit"
+					}
+				},
+				{
+					"type": "Input.Number",
+					"id": "Input.Number_id",
+					"label": "Input.Number_label",
+					"max": 9.5,
+					"min": 3.5,
+					"placeholder": "Input.Number_placeholder",
+					"value": 4.5,
+					"isRequired": true
+				},
+				{
+					"type": "Input.Date",
+					"id": "Input.Date_id",
+					"label": "Input.Date_label",
+					"min": "8/1/2018",
+					"max": "1/1/2020",
+					"placeholder": "Input.Date_placeholder",
+					"value": "8/9/2018"
+				},
+				{
+					"type": "Input.Time",
+					"id": "Input.Time_id",
+					"label": "Input.Time_label",
+					"min": "10:00",
+					"max": "17:00",
+					"value": "13:00",
+					"placeholder": "Input.Time_placeholder",
+					"isRequired": true,
+					"errorMessage": "Input.Time.ErrorMessage"
+				},
+				{
+					"type": "Input.Toggle",
+					"id": "Input.Toggle_id",
+					"label": "Input.Toggle_label",
+					"title": "Input.Toggle_title",
+					"value": "Input.Toggle_on",
+					"valueOff": "Input.Toggle_off",
+					"valueOn": "Input.Toggle_on"
+				},
+				{
+					"type": "TextBlock",
+					"weight": "BoLdEr",
+					"size": "large",
+					"text": "Everybody's got choices"
+				},
+				{
+					"type": "Input.ChoiceSet",
+					"id": "Input.ChoiceSet_id",
+					"isMultiSelect": true,
+					"label": "Input.ChoiceSet_label",
+					"style": "compact",
+					"value": "Input.Choice2,Input.Choice4",
+					"choices": [
+						{
+							"type": "Input.Choice",
+							"title": "Input.Choice1_title",
+							"value": "Input.Choice1"
+						},
+						{
+							"type": "Input.Choice",
+							"title": "Input.Choice2_title",
+							"value": "Input.Choice2"
+						},
+						{
+							"type": "Input.Choice",
+							"title": "Input.Choice3_title",
+							"value": "Input.Choice3"
+						},
+						{
+							"type": "Input.Choice",
+							"title": "Input.Choice4_title",
+							"value": "Input.Choice4"
+						}
+					]
+				}
+			]
+		},
+		{
+			"type": "ActionSet",
+			"actions": [
+				{
+					"type": "Action.Submit",
+					"title": "ActionSet.Action.Submit",
+					"id": "ActionSet.Action.Submit_id",
+					"associatedInputs": "none",
+<<<<<<< HEAD
+					"tooltip": "tooltip"
+=======
+					"description": "This is a description"
+>>>>>>> 550e574fb (Rename Tooltip to Description)
+				},
+				{
+					"type": "Action.OpenUrl",
+					"title": "ActionSet.Action.OpenUrl",
+					"id": "ActionSet.Action.OpenUrl_id",
+<<<<<<< HEAD
+					"tooltip": "tooltip",
+=======
+					"description": "This is a description",
+>>>>>>> 550e574fb (Rename Tooltip to Description)
```

Comment:
looks like there might have been an incomplete merge -- there are a bunch of `.orig` files that still have merge markers in them...

Reply:
Fixed, although i think we're probably not going to take this at this point.